pyproject.toml

```diff
@@ -0,0 +1,112 @@
+[tool.poetry]
+name = "connexion"
+version = "2.0.dev0"
+description = "Connexion - API first applications with OpenAPI/Swagger"
+readme = "README.md"
+keywords = ["api", "swagger", "openapi"]
+license = "Apache-2.0"
+authors = [
+    "Robbe Sneyders <robbe.sneyders@gmail.com>",
+    "Ruwan Lambrichts <ruwan.lambrichts@ml6.eu>",
+    "Daniel Grossmann-Kavanagh <me@danielgk.com>",
+    "Henning Jacobs <henning.jacobs@zalando.de>",
+    "João Santos <joao.santos@zalando.de>",
+]
+maintainers = [
+    "Robbe Sneyders <robbe.sneyders@gmail.com>",
+    "Ruwan Lambrichts <ruwan.lambrichts@ml6.eu>",
+]
+repository = "https://github.com/spec-first/connexion"
+include = ["*.txt", "*.rst"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Internet",
+    "Topic :: Internet :: WWW/HTTP",
+    "Topic :: Internet :: WWW/HTTP :: HTTP Servers",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Typing :: Typed",
+]
+
+[tool.poetry.scripts]
+connexion = 'connexion.cli:main'
+
+[tool.poetry.dependencies]
+python = '^3.9'
+asgiref = ">= 3.4"
+httpx = ">= 0.23"
+inflection = ">= 0.3.1"
+jsonschema = ">=4.17.3"
+Jinja2 = ">= 3.0.0"
+python-multipart = ">= 0.0.5"
+PyYAML = ">= 5.1"
+requests = ">= 2.27"
+typing-extensions = ">= 4.6.1"
+werkzeug = ">= 2.2.1"
```

Comment:
Let's bump this to a version with the CVE fixes, since this was the main reason for this PR.
3.1.3 is the version used in the 3.9 workflow with passing tests.